### ISSUE TITLE
Non-mobile platform not deployed on NPM

### ISSUE THREAD

**Issue by Serfenia** [ded/bowser#126]
Hello,

In README.md is stated that you can check the platform of the device like:
```
bowser.mac; // for mac
bowser.windows // for windows
bowser.linux // for linux
```

Though it is never deployed on npm. The installed package does not have the checks for platform like you have in your `./src/bowser.js` file on Github. 

I also noticed your `./src/bowser.js` and `./bowser.js` differ on Github.

Regards,
Patrick

**Comment by lancedikson** [comment]
@Serfenia, Hey, Patrick. Thank you for the answer. Now these files are synced already. Today have been released 1.1.0 in github, npm and bower.

**Comment by Serfenia** [comment]
Thanks!
